### ISSUE TITLE
reproduce

### ISSUE THREAD

**Issue by zfjsail** [anaivebird/kdd_pst#1]
Hi,

Please provide your model checkpoints.

Please provide the inference scripts on TEST data and your training scripts on the training data separately. The current codebase seems work on validation set.

**Comment by anaivebird** [comment]
模型checkpoint已经上传，训练脚本为bert.py，推理脚本为run_test.py
见：https://github.com/anaivebird/kdd_pst

https://drive.google.com/file/d/1ZHDV4BONcMFPzbjIjnnRCg456Rpod2rs/view?usp=sharing

下载后放到：本项目目录/out/kddcup/scibert/pytorch_model.bin
